### PR TITLE
blueutil: Drop dependency on Xcode, use makefile

### PR DESCRIPTION
**Pull request by chdiza**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Version 1.1.0 introduced, and 1.1.2 repaired, a makefile that allows the user to avoid Xcode altogether.    Use it and drop the ":build" dep on Xcode.

### CODE REVIEW

File: Formula/blueutil.rb [hunk 1]
```diff
@@ -13,12 +13,9 @@ class Blueutil < Formula
...
   end
 
-  depends_on :xcode => :build
-
   def install
-    # Set to build with SDK=macosx10.6, but it doesn't actually need 10.6
-    xcodebuild "SDKROOT=", "SYMROOT=build"
-    bin.install "build/Release/blueutil"
+    system "make"
+    bin.install "blueutil"
```
Comment (ilovezfs) [inline Formula/blueutil.rb]: can we convince them to add an install target too ...

Comment (chdiza) [inline Formula/blueutil.rb]: There is one; I'll need to make an inreplace, though.  I'll add this later today, I can't do it right this second.

Comment (chdiza) [inline Formula/blueutil.rb]: All that gets installed is the binary.  Theres no manpage.  Seems simpler to just `bin.install`, but whatever.

Comment (ilovezfs) [inline Formula/blueutil.rb]: Preference for `make install` over `bin.install` is one of the tenets of Brewism.

**Comment by ilovezfs** [comment]
I think you can
```
system "make", "DESTDIR=#{prefix}", install
```

**Comment by chdiza** [comment]
> I think you can
> system "make", "DESTDIR=#{prefix}", "install"

No, I think that will hardcode `/usr/local`, and then there'd have to be an inreplace to fix that.

**Comment by chdiza** [comment]
(The Makefile hardcodes `/usr/local`, and setting `DESTDIR` to our `#{prefix}` doesn't undo the hardcoding.)

**Comment by ilovezfs** [comment]
Ah, you can just do `prefix=#{prefix}` but why are they only echoing the command not executing it?

**Comment by ilovezfs** [comment]
```
    inreplace "Makefile", "@echo $(INSTALL_PROGRAM)", "$(INSTALL_PROGRAM)"
    bin.mkpath
    system "make", "prefix=#{prefix}", "install"
```
The first two lines should not be necessary and are reportable bugs.

**Comment by chdiza** [comment]
It's not worth the effort to me when `bin.install` seems to exist for exactly this kind of case.

Never mind.

**Pull request closed without merging**